packages/kit/src/runtime/client/client.js

```diff
@@ -146,6 +146,19 @@ function native_navigation(url) {
 	return new Promise(() => {});
 }
 
+/**
+ * Checks whether a service worker is registered, and if it is,
+ * tries to update it.
+ */
+async function update_service_worker() {
+	if ('serviceWorker' in navigator) {
+		const registration = await navigator.serviceWorker.getRegistration(base || '/');
```

Comment:
> If this parameter is not provided, the current client's URL will be used by default.
https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/getRegistration

Do we consider leaving the parameters empty?